### PR TITLE
Feature/code cleanups

### PR DESCRIPTION
**Pull request by lyricnz**
Just Ideas

### CODE REVIEW

File: cointop/config.go [hunk 1]
```diff
@@ -262,22 +237,12 @@ func (ct *Cointop) ConfigToToml() ([]byte, error) {
...
-	var columnsIfc interface{} = ct.State.portfolioTableColumns
-	portfolioIfc["columns"] = columnsIfc
-
-	var currencyIfc interface{} = ct.State.currencyConversion
-	var defaultViewIfc interface{} = ct.State.defaultView
-	var defaultChartRangeIfc interface{} = ct.State.defaultChartRange
-	var colorschemeIfc interface{} = ct.colorschemeName
-	var refreshRateIfc interface{} = uint(ct.State.refreshRate.Seconds())
-	var cacheDirIfc interface{} = ct.State.cacheDir
+	portfolioIfc["columns"] = ct.State.portfolioTableColumns
```
Comment (miguelmota) [inline cointop/config.go:240]: can `portfolioIfc` be constructed similar to `favoritesMapIfc` or `tableMapIfc`?

File: cointop/table_header.go [hunk 2]
```diff
@@ -26,97 +26,97 @@ type HeaderColumn struct {
 
 // HeaderColumns are the header column widths
 var HeaderColumns = map[string]*HeaderColumn{
-	"rank": &HeaderColumn{
+	"rank": {
```
Comment (miguelmota) [inline cointop/table_header.go:29]: this is a really nice clean up improvement

File: cointop/config.go [hunk 3]
```diff
@@ -242,8 +218,7 @@ func (ct *Cointop) ConfigToToml() ([]byte, error) {
 		"names":   favoritesIfc,
 	}
 
-	var favoritesColumnsIfc interface{} = ct.State.favoritesTableColumns
-	favoritesMapIfc["columns"] = favoritesColumnsIfc
+	favoritesMapIfc["columns"] = ct.State.favoritesTableColumns
```
[the remark this answers is not in the document]
Comment (simon-anz) [reply cointop/config.go]: done

File: cointop/config.go (hunk 1 above)
Comment (simon-anz) [reply cointop/config.go]: done

**Comment by miguelmota** [comment]
Ready for approval

**Review by miguelmota — APPROVED** [review]

**Pull request merged**